docs/source/markdown/podman-run.1.md

```diff
@@ -1234,7 +1234,7 @@ you'd like to connect instead, as in:
 $ podman run -a stdin -a stdout -i -t fedora /bin/bash
 ```
 
-## Sharing IPC between containers
+### Sharing IPC between containers
```

Comment:
Can you check the `podman create` manpage to see if it also has this issue?

Reply:
`podman create` has only a few examples, and this IPC one isn't one of them. :+1: